Formula/mupdf-tools.rb

```diff
@@ -1,10 +1,23 @@
 class MupdfTools < Formula
   desc "Lightweight PDF and XPS viewer"
   homepage "https://mupdf.com/"
-  url "https://mupdf.com/downloads/mupdf-1.11-source.tar.gz"
-  sha256 "209474a80c56a035ce3f4958a63373a96fad75c927c7b1acdc553fc85855f00a"
+  revision 1
   head "https://git.ghostscript.com/mupdf.git"
 
+  stable do
+    url "https://mupdf.com/downloads/mupdf-1.11-source.tar.gz"
+    sha256 "209474a80c56a035ce3f4958a63373a96fad75c927c7b1acdc553fc85855f00a"
+
+    # Upstream already. Remove on next stable release.
+    patch do
+      url "https://mirrorservice.org/sites/ftp.debian.org/debian/pool/main/m/mupdf/mupdf_1.11+ds1-1.1.debian.tar.xz"
```

Comment:
url & mirror

Reply:
Sorry, I was having some tediously slow internet speeds & just wanted to get something pushed 😓. Will sort this shortly.